tests/test_elastic_emt.py

```diff
@@ -17,7 +17,7 @@ def test_calc_elastic(self):
             fit_order=2
         )
         structure_dict = calculator.generate_structures()
-        result_dict = evaluate_with_ase(task_dict=structure_dict, ase_calculator_class=EMT)
+        result_dict = evaluate_with_ase(structure_dict, ase_calculator=EMT())
```

Comment:
Is it necessary to instantiate the ASE calculator? I am not sure if all ASE calculators support multiple evaluations, but that is something we can test.

Reply:
It's not necessary, but if they will support it then it's preferable. This way we don't have extra args and kwargs kicking around. It's a small gain though, so if it breaks for one of the calculators we can switch back.

Reply:
I somewhat prefer passing the instances, because this is a very well established [OOP pattern](https://en.wikipedia.org/wiki/Dependency_injection).  I'm not aware of ASE calculators that cannot be used multiple times, though there are some where this doesn't make sense (e.g. SinglePointCalculator).  There might be an issue with file based calculators overwriting the results of previous runs.  If we think that's an issue, I would also pass the structure label to the 'inner evaluator' that @liamhuber defines here and then just do `calc.set_label(...)` inside the generic ASE function.

Reply:
Nice. Sounds like we can leave it as is and patch later if/when needed.